routes/artifacts.js

```diff
@@ -190,6 +190,18 @@ api.declare({
         details.bucket  = this.privateBucket.bucket;
       }
       details.prefix    = [taskId, runId, name].join('/');
+      if (input.contentMD5 || input.contentSHA256) {
+        // Remove this check, when contentMD5 and contentSHA256 are required
+        if (!input.contentMD5 || !input.contentSHA256) {
```

Comment:
Might be a stupid simple test, but any value in having a test to make sure both are present?